src/sentry/models/exporteddata.py

```diff
@@ -57,18 +57,17 @@ def status(self):
         else:
             return ExportStatus.Valid
 
-    @property
-    def date_expired_string(self):
-        if self.date_expired is None:
-            return None
-        return self.date_expired.strftime("%-I:%M %p on %B %d, %Y (%Z)")
-
     @property
     def payload(self):
         payload = self.query_info.copy()
         payload["export_type"] = ExportQueryType.as_str(self.query_type)
         return payload
 
+    @classmethod
+    def format_date(cls, date):
```

Comment:
@leeandher Nit: I don't think this needs to be a class method as there is no dependency on `cls`

Reply:
I just did this so in the tests the line became:
```python
ExportedData.format_date(self.data_export.date_added)
```
instead of 
```python
self.data_export.format_date(self.data_export.added)
```
since at that point it seems better to just make the three methods (`format_date_added`, `format_date_expired`, `format_date_finished`)

Reply:
I see where i goofed, nvm